app/src/main/res/values/strings.xml

```diff
@@ -12,6 +12,7 @@
   <string name="languageJa">Japanese</string>
   <string name="languageKo">Korean</string>
   <string name="languageRu">Russian</string>
+  <string name="languageCz">Czech</string>
```

Comment:
Note that `languageCz` is currently unnecessary, it's for languages in `Language` setting, these are languages for which a custom display mode is implemented (verbose mode, etc).

Reply:
ok, I create separate PR for that